grafana/overlays/moc/balrog/opf-monitoring/dashboards/thanos/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
We wont have thanos deployment in balrog,as we are using the openshift-monioring
please remove the whole sub-dir thanos.